sites/fast-website/src/public/index.html

```diff
@@ -45,7 +45,7 @@
               <fast-badge slot="badge" class="section-badge">ADAPTIVE UI SYSTEM, UTILITIES, & TOOLS</fast-badge>
               <h1 class="section-heading-hero">The adaptive interface system for modern web experiences</h1>
               <p slot="body">Interfaces bult with FAST adapt to your design system and can be used with any modern UI Framework by leveraging industry standard Web Components.</p>
-              <fast-button slot="action" appearance="accent">Get started</fast-button>
+              <fast-button slot="action" appearance="accent" href="/docs/fast-foundation/getting-started">Get started</fast-button>
```

Comment:
I think this one makes sense to have direct to the getting started, but the one above should probably go to the docs intro. Thoughts?

Reply:
I'll have another PR to cover the rest of the page's links.